src/components/highlight-cards/highlight-cards.js

```diff
@@ -19,88 +20,87 @@ function HighlightCards({
   nopadtop,
   hasCustomCols,
 }) {
+  const styles =
+    "highlight-cards" +
+    `${background ? ` bg-${background}` : ""}` +
+    `${nospace ? "" : "  py-5 py-lg-7"}` +
+    `${nopadtop ? "  no-pad-top" : ""}`;
 
-  const styles = 'highlight-cards'
-	+ `${background ? ` bg-${background}` : ''}`
-  + `${nospace ? '' : '  py-5 py-lg-7'}`
-  + `${nopadtop ? '  no-pad-top' : ''}`
+  let cardStyles = classNames("col-12 col-md-6 mb-3 mb-md-4", {
+    "col-lg-4": !col4,
```

Comment:
@Fupete btw, I kept this as it was, but it doesn't look right to begin with

Reply:
Seems right to me. I remember came from a classNames example. Are you sure?

Reply:
@Fupete this looks backwards, it should be `"col-lg-4"` is `col4` is truthy.

Reply:
They mean something different... col4 = 4 columns layout in our templates. col-lg-4 mean 4 bootstrap columns (of 12 total) largeness for each column = 3 columns layout. I know, weird but it works.